suite-native/app/src/App.tsx

```diff
@@ -22,19 +23,30 @@ import { useFormattersConfig } from './hooks/useFormattersConfig';
 const AppComponent = () => {
     const dispatch = useDispatch();
     const formattersConfig = useFormattersConfig();
+    const [isConnectInitialized, setIsConnectInitialized] = useState(false);
 
     useEffect(() => {
         const initActions = async () => {
-            await dispatch(connectInitThunk()).unwrap();
-            await dispatch(initBlockchainThunk()).unwrap();
-            /* Invoke reconnect manually here because we need to have fiat rates initialized
-             immediately after the app is loaded.
-             */
-            await dispatch(reconnectBlockchainThunk('btc')).unwrap();
+            try {
+                await dispatch(connectInitThunk()).unwrap();
+                setIsConnectInitialized(true);
+                await dispatch(initBlockchainThunk()).unwrap();
+                /* Invoke reconnect manually here because we need to have fiat rates initialized
+                 * immediately after the app is loaded.
+                 */
+                await dispatch(reconnectBlockchainThunk('btc')).unwrap();
+            } catch (error) {
+                Alert.alert('Error', error?.message ?? 'Unknown error');
```

Comment:
I not sure if this will actually show because it will be hidden under splash screen